docs/ldml/tr35.md

```diff
@@ -1350,38 +1350,37 @@ Hybrid locales have intermixed content from 2 (or more) languages, often with on
 
-The value of -t- is a full _[unicode_language_id](#unicode_language_id)_, and can contain subtags for region where it is important to include it, as in the following. The value can also include the script, although that is is not normally included: the only instance where it should be is where the content of the source text varies by script. So because zh-Hant has different vocabulary and expressions, it could make sense to have en-t-zh-hant to make that distinction.
+The value of -t- is a full _[unicode_language_id](#unicode_language_id)_, and can contain subtags for region where it is important to include it, as in the following. The value can also include the script, although that is is not normally included: the only instance where it should be is where the content of the source text varies by script. So because zh-Hant has different vocabulary and expressions, it could make sense to have en-t-zh-Hant to make that distinction.
 
 > Note: The default script for the language is computed without reference to the hybrid subtags. Thus the default script for 'ru' is “Cyrl”, no matter what the source is in the -t- tag.
 
-<!-- HTML: no header -->
-<table><tbody>
-<tr><td>ru-t-<u>en-h0-hybrid</u></td>   <td>Runglish</td>   <td>Russian with an admixture of American English</td></tr>
-<tr><td>ru-t-<u>en-gb-h0-hybrid</u></td>   <td>Runglish</td>   <td>Russian with an admixture of British English</td></tr>
-<tr><td>ru-Latn-t<u>-en-gb-h0-hybrid</u></td>   <td>Runglish</td>   <td>Russian with an admixture of British English in Latin script</td></tr>
-<tr><td>en-t-<u>zh-h0-hybrid</u></td>   <td>Chinglish</td>   <td>American English with an admixture of Chinese (Simplified Mandarin Chinese)</td></tr>
-<tr><td>en-t-<u>zh-Hant-h0-hybrid</u></td>   <td>Chinglish</td>   <td>American English with an admixture of Chinese (Traditional Mandarin Chinese)</td></tr>
-</tbody></table>
+|Hybrid locale code		| Hybrid name	| Description |
+|-------------------------------|---------------|-------------|
+|ru-t-***en-h0-hybrid***	| Runglish	| Russian with an admixture of American English|
+|ru-t-***en-gb-h0-hybrid***	| Runglish	| Russian with an admixture of British English|
+|ru-Latn-t-***-en-gb-h0-hybrid***| Runglish	| Russian with an admixture of British English in Latin script|
+|en-t-***zh-h0-hybrid***	| Chinglish	| American English with an admixture of Chinese (Simplified Mandarin Chinese)|
+|en-t-***zh-Hant-h0-hybrid***	| Chinglish	| American English with an admixture of Chinese (Traditional Mandarin Chinese)|
 
 Should there ever be strong need for hybrids of more than two languages or for other purposes such as hybrid languages as the source of translated content, additional structure could be added.
```

Comment:
CLDR-13370 is asking for a change of this sentence as well:
> As a native English speaker, this was ambiguous to me (and implied that adding more structure — e.g. `-h0-` tags— this could be done.  I've submitted a ticket for enabling such things, but until then, this sentence should state unequivocally “It is not currently possible to repreesent hybrids of more than two languages, nor enable hybrid languages as the source of translated content.  Please file a ticket if you need this support”.

@macchiati do you agree? Should we change it as suggested, or reword it?